src/main/java/io/stargate/sgv2/jsonapi/api/model/command/JsonDefinition.java

```diff
@@ -0,0 +1,25 @@
+package io.stargate.sgv2.jsonapi.api.model.command;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * Value type enclosing JSON-decoded representation of something to resolve into fully definition,
+ * such as filter or sort clause. Needed to let Quarkus/Jackson decode textual JSON into
+ * intermediate form ({@link JsonNode}) from which actual deserialization and validation can be
+ * deferred until we have context we need
+ */
+public abstract class JsonDefinition {
```

Comment:
@amorton Adding this as shareable base definition for `FilterSpec` (or whatever we call it) and similar intermediate sources. The reason for base class over interface is to allow field for actual `JsonNode`. And the reason to require sub-class is to allow/require Swagger annotation(s) (`@Schema`); also helps slightly with documentation/type-safety.